src/main/java/dwbh/api/domain/input/UserGroupInput.java

```diff
@@ -17,46 +17,47 @@
  */
 package dwbh.api.domain.input;
 
-import java.util.UUID;
+import dwbh.api.domain.Group;
+import dwbh.api.domain.User;
 
 /**
  * UserGroup input. It contains the ids for a user and a group
  *
  * @since 0.1.0
  */
 public class UserGroupInput {
-  private final UUID userId;
-  private final UUID groupId;
+  private final User user;
```

Comment:
Why an input type receives the whole domain class ? I've seen that in the services you end up doing `input.getGroup().getId()` and `input.getUser().getId()`. So I don't see the point of doing this.